python/cudf/cudf/core/series.py

```diff
@@ -1245,21 +1245,21 @@ def logical_and(self, other):
             "Series.logical_and is deprecated and will be removed.",
             FutureWarning,
         )
-        return self._binaryop(other, "l_and").astype(np.bool_)
+        return self._binaryop(other, "__l_and__").astype(np.bool_)
```

Comment:
Can we use `__logical_and___` instead?

Reply:
Let's wait to address this in a follow-up PR. This particular method is deprecated anyway, so I'm not sure that this change is worth it, but in the next PR when I use this mixin for ColumnBase I'll also have to rename a lot of these strings again for consistency.